src/main/java/tests/TestCase.java

```diff
@@ -1,30 +1,30 @@
 package tests;
-import loginPage.Homepage;
+import loginpage.HomePage;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.firefox.FirefoxDriver;
 import org.testng.annotations.AfterTest;
 import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 
-public class testCase {
+public class TestCase {
     WebDriver driver = new FirefoxDriver();
-    Homepage page;
+    HomePage loginhomepage;
```

Comment:
so this should probably just be `HomePage home` => with loginhomepage, it doesn't clearly state whether you are on the login page or the home page when reading it in the code.

Also, the name should be in camel case like `loginHomePage` if you were going to name it like that